lib/html.rb

```diff
@@ -73,7 +73,13 @@ def self.scope_help(scope)
   end
 
   def self.diff_to_object(diff)
-    diff.encode!("UTF-8", invalid: :replace, undef: :replace)
+    diff.encode!(Encoding.find('ASCII'), {
+      invalid: :replace,
+      undef: :replace,
+      replace: '',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.